root/edit/details/AddRemoveAlias.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * @flow
+ * Copyright (C) 2019 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+import React from 'react';
+
+import bracketed from '../../static/scripts/common/utility/bracketed';
+import formatEntityTypeName from '../../static/scripts/common/utility/formatEntityTypeName';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import isDateEmpty from '../../static/scripts/common/utility/isDateEmpty';
+import DescriptiveLink from '../../static/scripts/common/components/DescriptiveLink';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import locales from '../../static/scripts/common/constants/locales';
+import isolateText from '../../static/scripts/common/utility/isolateText';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type AliasEntityTypeT = $ElementType<AliasEntityTypeT, 'entityType'>;
+
+type DisplayDataT = {
```

Comment:
I'd prefer creating an `AddRemoveAliasEditT` type using the definition from `Props` below, and inlining the `display_data` definition there. Since we may want to refer to these edits outside of props. See root/edit/details/AddAnnotation.js for example.